tendermint/src/vote.rs

```diff
@@ -159,6 +158,7 @@ impl Vote {
 }
 
 /// Default trait. Used in tests.
+// FIXME: Does it need to be in public crate API? If not, replace with a helper fn in crate::test?
```

Comment:
To expand on this: it's probably not useful for non-test API users to have a `Default` impl for `Vote`, and there is potential for inadvertent misuse that is hard to audit for.

For test users, a dedicated helper fn would serve just as well, unless there are some commonly used cases that are more ergonomic with a `Default` impl. The dedicated function will be easy to find in code.

Reply:
Agreed, a dedicated helper fn sounds like the way to go. As far as I can tell this `Default` instance is only used in this very test (both directly via `Vote::default()` and indirectly via `..Default::default`): https://github.com/informalsystems/tendermint-rs/blob/9690eca1b97685f699da58c1d9f3f8a85d5c33a9/tendermint/src/vote/sign_vote.rs#L277

Reply:
Tracked in https://github.com/informalsystems/tendermint-rs/issues/1025